spec/components/all_components_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe "AllComponents", type: :controller do
+  Dir.glob("app/views/components/*.erb").each do |filename|
+    template = filename.split("/").last
+    component_name = template.sub("_", "").sub(".html", "").sub(".erb", "").gsub("-", "_")
+
+    context(component_name) do
```

Comment:
This is a very boring comment, but I've seen some people use `context` and some people use `describe`. I think we should standardise on one. Or an at least decide  when one should be used over the other.

Reply:
I've detailed my use of describe vs context in the PR description. (Basically, describe when it's top level or the comment is just a method name, context otherwise, which I think makes most sense semantically)

Reply:
So I think then that this should be a `describe` rather than a `context`?

I've generally thought that `describe` is used for "things" e.g. method names, routes, etc and `context` is used to describe the circumstances of the test, e.g. a set of tests for postcode outcomes, could have a context of "in England".

But this is testing a components, which I think is a "thing" not a "circumstance".

Reply:
Oh, I see. We're agreeing on style, you're just pointing out there's a mistake here. Yes, it should be a describe.